plugins/aea-test-autonomy/aea_test_autonomy/docker/registries.py

```diff
@@ -51,71 +49,29 @@
 class RegistriesDockerImage(DockerImage):
     """Spawn a local Ethereum network with deployed registry contracts, using HardHat."""
 
-    third_party_contract_dir: Path
-
     def __init__(
         self,
         client: docker.DockerClient,
-        third_party_contract_dir: Path,
         addr: str = DEFAULT_HARDHAT_ADDR,
         port: int = DEFAULT_HARDHAT_PORT,
     ):
         """Initialize."""
         super().__init__(client)
         self.addr = addr
         self.port = port
-        self.third_party_contract_dir = third_party_contract_dir
 
     @property
     def tag(self) -> str:
         """Get the tag."""
-        return "node:16.7.0"
-
-    def _build_command(self) -> List[str]:
-        """Build command."""
-        cmd = ["run", "hardhat", "node", "--port", str(self.port)]
-        return cmd
-
-    def _update_config_hash(
-        self,
-    ) -> None:
-        """Updated config hash in the registry config."""
-
-        base_snapshot_file = (
-            self.third_party_contract_dir
-            / REGISTRIES_CONTRACTS_DIR
-            / "scripts"
-            / "mainnet_snapshot.json"
-        )
-        snapshot_data = json.loads(base_snapshot_file.read_text())
-        snapshot_data["serviceRegistry"]["configHashes"] = [
-            DEFAULT_SERVICE_CONFIG_HASH,
-        ]
-        snapshot_file = (
-            self.third_party_contract_dir / REGISTRIES_CONTRACTS_DIR / "snapshot.json"
-        )
-        snapshot_file.write_text(json.dumps(snapshot_data))
```

Comment:
We need to make sure that the image has support for the configuration require in this repo. This can be done on `autonolas-registries` repo side.

Reply:
I already have a Dockerfile that accepts this hash as an env var. It defaults back to what this here defaults to.